alerts/lib/alerttask.py

```diff
@@ -393,7 +393,7 @@ def createAlertDict(self, summary, category, tags, events, severity='NOTICE', ur
         for e in events:
             alert['events'].append({
                 'documentindex': e['_index'],
-                'documenttype': e['_type'],
+                'documenttype': e['type'],
```

Comment:
in my head I feel like maybe I should remove documenttype from this stanza, and add t his later as simply 'type' so that there is no confusion between _type: _doc and type: <alert type>